app/components/machine/driver-amazonec2/component.js

```diff
@@ -318,29 +422,79 @@ let INSTANCE_TYPES = [
   },
 
   {
-    group: 'R5D - High Memory Optimized & Local Storage',
+    group: 'R5d - High Memory Optimized & Local Storage',
     name:  'r5d.large'
   },
   {
-    group: 'R5D - High Memory Optimized & Local Storage',
+    group: 'R5d - High Memory Optimized & Local Storage',
     name:  'r5d.xlarge'
   },
   {
-    group: 'R5D - High Memory Optimized & Local Storage',
+    group: 'R5d - High Memory Optimized & Local Storage',
     name:  'r5d.2xlarge'
   },
   {
-    group: 'R5D - High Memory Optimized & Local Storage',
+    group: 'R5d - High Memory Optimized & Local Storage',
     name:  'r5d.4xlarge'
   },
   {
-    group: 'R5D - High Memory Optimized & Local Storage',
+    group: 'R5d - High Memory Optimized & Local Storage',
     name:  'r5d.12xlarge'
   },
   {
-    group: 'R5D - High Memory Optimized & Local Storage',
+    group: 'R5d - High Memory Optimized & Local Storage',
     name:  'r5d.24xlarge'
   },
+  
+  {
+    group: 'R5a - High Memory Optimized AMD Epyc',
+    name:  'r5a.large'
+  },
+  {
+    group: 'R5a - High Memory Optimized AMD Epyc',
+    name:  'r5a.xlarge'
+  },
+  {
+    group: 'R5a - High Memory Optimized AMD Epyc',
+    name:  'r5a.2xlarge'
+  },
+  {
+    group: 'R5a - High Memory Optimized AMD Epyc',
+    name:  'r5a.4xlarge'
+  },
+  {
+    group: 'R5a - High Memory Optimized AMD Epyc',
+    name:  'r5a.12xlarge	'
+  },
+  {
+    group: 'R5a - High Memory Optimized AMD Epyc',
+    name:  'r5a.24xlarge	'
```

Comment:
The white space here is matching the `name:  'r5.24xlarge	'` instances, for consistency. Maybe this whitespace is a pre-existing issue?